runconfig/opts/parse.go

```diff
@@ -100,6 +101,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 	cmd.Var(&flVolumes, []string{"v", "-volume"}, "Bind mount a volume")
 	cmd.Var(&flTmpfs, []string{"-tmpfs"}, "Mount a tmpfs directory")
 	cmd.Var(&flLinks, []string{"-link"}, "Add link to another container")
+	cmd.Var(&flAliases, []string{"-network-alias"}, "Add network scoped alias for the container")
```

Comment:
I suggest renaming this to `--net-alias` to make it more convenient and consistent with `--net`